cypress/fixtures/commonComponents/pod.json

```diff
@@ -26,20 +26,20 @@
   "paddingExtraLarge": {
     "padding": "extra-large"
   },
-  "podTypePrimary": {
-    "podType": "primary"
+  "variantPrimary": {
```

Comment:
can You please also clear up this file after deleting tests from `pod.feature` file?